apps/src/templates/sectionProgressV2/InviteToV2ProgressModal.jsx

```diff
@@ -34,11 +34,13 @@ function InviteToV2ProgressModal({
   const [invitationOpen, setInvitationOpen] = React.useState(false);
 
   React.useEffect(() => {
-    const timeSinceInvitationLastDelayed = () => {
+    const numDaysSinceInvitationLastDelayed = () => {
+      const MILLISECONDS_IN_ONE_DAY = 1000 * 3600 * 24;
```

Comment:
Nit: move this constant outside the `InviteToV2ProgressModal` function (to the top of the file).